lib/git.js

```diff
@@ -1,92 +1,108 @@
-const shell = require('shelljs');
-const fs = require('fs');
-const path = require('path');
-const chalk = require('chalk');
+const shell = require("shelljs");
+const fs = require("fs");
+const path = require("path");
+const chalk = require("chalk");
 
 // utils
 const generateRemoteLinkSsh = (username, projectName) => {
-  return `git@github.com:${username}/${projectName}.git`
-}
+  return `git@github.com:${username}/${projectName}.git`;
+};
 const generateRemoteLinkHttps = (username, projectName) => {
-  return `https://github.com/${username}/${projectName}.git`
-}
+  return `https://github.com/${username}/${projectName}.git`;
+};
 const generateRemoteLink = (username, projectName, connectionType) => {
-  if (connectionType === 'https') {
+  if (connectionType === "https") {
     return generateRemoteLinkHttps(username, projectName);
   } else {
     return generateRemoteLinkSsh(username, projectName);
   }
-}
+};
 
 // Native actions
 const init = () => {
-  shell.exec('git init');
-}
+  shell.exec("git init");
+};
 
 const addRemote = (name, remote) => {
   shell.exec(`git remote add ${name} ${remote}`);
-}
+};
 
 const removeRemote = (name) => {
   shell.exec(`git remote remove ${name}`);
-}
+};
 
 const addSubtree = (subtree) => {
-  shell.exec(`git add ${subtree} && git commit -m "initial ${subtree} subtree commit"`);
-}
+  shell.exec(
+    `git add ${subtree} && git commit -m "initial ${subtree} subtree commit"`
+  );
+};
 
 const addCommitPushMaster = (commitMsg) => {
-  shell.exec(`git add . && git commit -m "${commitMsg}" && git push origin master`, {silent: true})
-}
+  shell.exec(`git add .`, { silent: true });
+  shell.exec(`git commit -m"${commitMsg}"`, {
+    silent: true,
+  });
+  shell.exec(`git push origin master`, {
+    silent: true,
+  });
+};
 
 // used mainly for pushing to gh-pages branch
 const pushSubtreeToOrigin = (subtree, branch) => {
-  if (branchDoesExistAtOrigin('gh-pages')) {
-    console.log('gh-pages already exists at origin')
+  if (branchDoesExistAtOrigin("gh-pages")) {
+    console.log("gh-pages already exists at origin");
   }
   shell.exec(`git subtree push --prefix ${subtree} origin ${branch}`);
-}
+};
 
 // Custom actions
 
-const createOrOverrideRemoteOrigin = (username, projectName, connectionType) => {
+const createOrOverrideRemoteOrigin = (
+  username,
+  projectName,
+  connectionType
+) => {
   // git init if needed
-  if(!hasGitInitialized()) {
-    console.log(chalk.cyanBright('Intializing git...'));
+  if (!hasGitInitialized()) {
+    console.log(chalk.cyanBright("Intializing git..."));
```

Comment:
[no action needed] i would really like to refactor all this code to isolate git calls into a service of their own. this file has way too much business logic.